po/ja.po

```diff
@@ -17976,12 +17981,18 @@ msgid ""
 "you must know how many futures you will have at compile time. This is "
 "currently in the `futures` crate, soon to be stabilised in `std::future`."
 msgstr ""
+"複数の互いに素な型のfutureに対しては、`std::future::join!`を利用できます。し"
+"かし、いくつのfutureがコンパイル時に存在しているのかを把握しておく必要があり"
+"ます。これは現在`future`クレートにありますが、近いうちに`std::future`に統合さ"
```

Comment:
nit: the name of the crate seems to be with 's', so `futures` 😄

Reply:
Thanks! Seems like I dropped `s` without noticing.. :)